go/flags/endtoend/vtctld.txt

```diff
@@ -79,8 +84,6 @@ Usage of vtctld:
       --proxy_tablets                                                    Setting this true will make vtctld proxy the tablet status instead of redirecting to them
       --purge_logs_interval duration                                     how often try to remove old logs (default 1h0m0s)
       --remote_operation_timeout duration                                time to wait for a remote operation (default 15s)
-      --s2a_enable_appengine_dialer                                      If true, opportunistically use AppEngine-specific dialer to call S2A.
-      --s2a_timeout duration                                             Timeout enforced on the connection to the S2A service for handshake. (default 3s)
```

Comment:
Any idea whether we actually need these flags?
Looks like they were being pulled in because of an indirect dependency that I can see in go.mod
```
github.com/google/s2a-go v0.1.3 // indirect
```

Reply:
https://github.com/google/s2a-go/issues/107